x-pack/plugins/upgrade_assistant/__jest__/client_integration/es_deprecations/deprecations_list.test.ts

```diff
@@ -66,29 +64,31 @@ describe('ES deprecations table', () => {
 
   it('refreshes deprecation data', async () => {
     const { actions } = testBed;
-    const totalRequests = server.requests.length;
 
     await actions.table.clickRefreshButton();
 
     const mlDeprecation = esDeprecationsMockResponse.deprecations[0];
     const reindexDeprecation = esDeprecationsMockResponse.deprecations[3];
 
     // Since upgradeStatusMockResponse includes ML and reindex actions (which require fetching status), there will be 4 requests made
-    expect(server.requests.length).toBe(totalRequests + 4);
-    expect(server.requests[server.requests.length - 4].url).toBe(
-      `${API_BASE_PATH}/es_deprecations`
+    expect(mockEnvironment.httpSetup.get).toHaveBeenCalledWith(
```

Comment:
If we are to refactor out `mockResponse` into `kbn-test` I think would be nice to have some sort of mechanism to access an array of requests made like what sinnon provides, given that you can choose what to match against rather than having to have an ordered list of `expect get toHaveBeenCalledWith`